interface/main/calendar/add_edit_event.php

```diff
@@ -1072,6 +1091,14 @@ function sel_patient() {
     dlgopen('find_patient_popup.php', 'findPatient', 650, 300, '', title);
 }
 
+// This invokes the cancellation popup.
```

Comment:
Just learning about this stuff, so will be good to ensure @adunsulag agrees with me on this.
Should not need to qualify whether to dispatch this event (the module that is listening can decide what to do). This is basically to render javascript on the add_edit_event script, so would make this more generic with something like the following (and send the $row so the listener/module can decide what to do):
```
$eventDispatcher->dispatch(ApptRenderEvent::RENDER_JAVASCRIPT, new ApptRenderEvent($row), 10);
```
(check out the dispatches in the demographics.php to get an idea for this where is offering custom rendering in different places in that script)

Reply:
I agree, just passing the row data and leaving it generic will be way better and allows more customizations than just telehealth.